docs/en/operations/settings/settings.md

```diff
@@ -979,6 +979,41 @@ Error count of each replica is capped at this value, preventing a single replica
 - [Table engine Distributed](../../operations/table_engines/distributed.md)
 - [`distributed_replica_error_half_life`](#settings-distributed_replica_error_half_life)
 
+
+## distributed_directory_monitor_sleep_time_ms {#distributed_directory_monitor_sleep_time_ms}
+
+Interval of data sending by the [Distributed](../table_engines/distributed.md) table engine. In case of any errors grows exponentially.
```

Comment:
```suggestion
Base interval of data sending by the [Distributed](../table_engines/distributed.md) table engine. Actual interval grows exponentially in case of any errors.
```